tests/tests_end_to_end/test_rules_flow.py

```diff
@@ -28,6 +28,21 @@ def test_excel_importer_to_yaml(self, data_regression: DataRegressionFixture) ->
         exported_rules = yaml.safe_load(exported_yaml_str)
         data_regression.check(exported_rules)
 
+    def test_excel_importer_to_yaml_new_endpoint(self, data_regression: DataRegressionFixture) -> None:
+        neat = NeatSession(verbose=False)
+
+        neat.data_model.read.excel(DOC_RULES / "information-architect-david.xlsx")
+
+        neat.convert()
+
+        exported_yaml_str = neat.to.yaml()
+
+        exported_rules = yaml.safe_load(exported_yaml_str)
+        data_regression.check(exported_rules)
```

Comment:
nit: Could have used custom base names so that you don't need to create a new file. I am not sure it is _better_ though 🤷‍♂️ 

https://pytest-regressions.readthedocs.io/en/latest/overview.html#generated-filenames

```suggestion
        data_regression.check(exported_rules, basename="test_excel_importer_to_yaml")
```